layers/libVkLayer_khronos_shader_object.map

```diff
@@ -2,6 +2,7 @@
   global:
     vkGetInstanceProcAddr;
     vkGetDeviceProcAddr;
+    vkEnumerateInstanceExtensionProperties;
```

Comment:
Shader object doesn't implement this function. As a result this will fail to build with Android NDK r26 which has stricter linker warnings.